### PR TITLE
Add group creation and user deletion endpoints

### PR DESCRIPTION
**Pull request by the4t4**
Added:

[POST] groups/create

- Request body: {groupname, timetable?}

[DELETE] users/

- Request body: { users: [{userid}] }

The delete endpoint is not complete as it does not remove the user from keycloak yet. Its best if @Mohido can complete that.

### CODE REVIEW

File: backend/routes/users.js [hunk 1]
```diff
@@ -183,6 +183,47 @@ router.put(
   }
 );
 
+/**
+ * Takes a list of users and deletes them in both the keycloak and database with random passwords.
```
Comment (Mohido) [inline backend/routes/users.js:187]: @the4t4  What? Is this a copy mistake? or it does do something with the passwords?

**Review by Mohido — APPROVED** [review]
Seems Legit!

**Comment by Mohido** [comment]
I will add the Keycloak deletion user deletion later on today and then we can merge this branch to the development.

File: backend/routes/users.js (hunk 1 above)
Comment (the4t4) [reply backend/routes/users.js:187]: Yes it's a copy mistake

**Pull request merged**